### DISCUSSION TITLE
Additional Functionality

### DISCUSSION THREAD

**Discussion by yaarite** [gdarko/digital-license-manager#41] [Ideas]
This plugin is wonderful and does almost everything I needed. I am using it to generate Warranties for a product (re-purposed) because there is no warranty registration plugins that are this efficient. The following 2 suggestions would be a relatively easy fix IMO. 

1) When a customer places an order for a Digital license they should not only get a copy of the order but also emailed a copy of the PDF license or at the very least a include a download link in the woocommerce order. Rather than having them log into the wordpress site after and manually download the license  from the link inside their account (way to many additional steps) . 

2) Also one more simple functionality in addition to the current Logo branding feature would be the ability to actually re-name the PDF or license Title . This would be a welcome additional functionality to white label the document. Instead of the word License at the top of the Document I would  like to be able to change it to Warranty or Prefix it with the Product name. 

At the very least tell us what file to edit in the plugin to accomplish the PDF Title Change

**Comment by gdarko** [comment]
Hey @yaarite 

Regarding 1. This requires some custom coding and can be done. I may include this feature in our PRO version in the future releases.

Regarding 2. As of Digital License Manager 1.6.0 you can change the PDF title as follows:
https://docs.codeverve.com/digital-license-manager/codex/snippets/how-to-change-the-license-certificate-title/

Best Regards,
Darko

**Comment by yaarite** [comment]
add_filter( 'dlm_license_certification_title', function ( $title ) {
    return 'Your title goes here...';
} );


Forgive me but which file in the plugin? or where on the functions php file. At the end of functions php? 


"To change the license certificate title from “License Certificate” to something else, place this code in your 

functions.php <---?????
or 
your plugins)."<---??????

You gave an either or answer and no specific file for the plugin or where to locate it. I was assuming it was a simple text output statement in a file just not sure what file?





On Friday, March 15, 2024 at 11:12:42 AM EDT, Darko Gjorgjijoski ***@***.***> wrote: 







Hey @yaarite

Regarding 1. This requires some custom coding and can be done. I may include this feature in our PRO version in the future releases.

Regarding 2. As of Digital License Manager 1.6.0 you can change the PDF title as follows:
https://docs.codeverve.com/digital-license-manager/codex/snippets/how-to-change-the-license-certificate-title/

Best Regards,
Darko

**Reply by gdarko** [reply]
Hey

In the theme's function.php

I mentioned plugins because some people add custom code in custom plugins

Sorry,
DG

**Comment by yaarite** [comment]
OK so just put that statement in the bottom of functions PHP and it will put out the title i want on the PDF license? 
Sorry i just dont want to crash the site by doing something wrong. I am a designer not a developer. 
I appreciate your help and will definitely buy the premium version






On Friday, March 15, 2024 at 01:13:17 PM EDT, Darko Gjorgjijoski ***@***.***> wrote: 







Hey

In the theme's function.php

I mentioned plugins because some people add custom code in custom plugins

Sorry,
DG

**Comment by gdarko** [comment]
Hey @yaarite ,

Sure, you need to change the text "Your title goes here", please keep the ' '. Just change the text and add the code in functions.php

Something like this:

```
add_filter( 'dlm_license_certification_title', function ( $title ) {
	return 'Warranty Information';
} );
```

**Comment by yaarite** [comment]
did it worked AWESOME thanks for the help



On Friday, March 15, 2024 at 03:39:07 PM EDT, Darko Gjorgjijoski ***@***.***> wrote: 







Hey @yaarite ,

Sure, you need to change the text "Your title goes here", please keep the ' '. Just change the text and add the code in functions.php

Something like this:
add_filter( 'dlm_license_certification_title', function ( $title ) {
    return 'Warranty Information';
} );

**Comment by gdarko** [comment]
Hey @yaarite, Great! I am glad that it worked. If you liked our support [leave us a review](https://wordpress.org/support/plugin/digital-license-manager/reviews/?filter=5).
